zaza/openstack/charm_tests/tempest/templates/tempest_v3.j2

```diff
@@ -141,6 +142,9 @@ member_role = load-balancer_member
 admin_role = Admin
 RBAC_test_type = none
 log_user_roles = False
+
+[loadbalancer_feature_enabled]
+prometheus_listener_enabled = False
```

Comment:
I'm glad you asked because this is actually not working as expected. There is a test that fails due to https://bugs.launchpad.net/snap-octavia-diskimage-retrofit/+bug/2027585 - I will add it to the exclude-list and a comment linked to the bug.